x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/activity_log_date_range_picker/index.tsx

```diff
@@ -6,118 +6,137 @@
  */
 
 import { useDispatch } from 'react-redux';
-import React, { memo, useCallback } from 'react';
+import React, { memo, useCallback, useState } from 'react';
 import styled from 'styled-components';
-import moment from 'moment';
-import { EuiFlexGroup, EuiFlexItem, EuiDatePicker, EuiDatePickerRange } from '@elastic/eui';
+import dateMath from '@elastic/datemath';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSuperDatePicker,
+  EuiSuperDatePickerRecentRange,
+} from '@elastic/eui';
 
-import * as i18 from '../../../translations';
 import { useEndpointSelector } from '../../../hooks';
 import { getActivityLogDataPaging } from '../../../../store/selectors';
+import { DEFAULT_TIMEPICKER_QUICK_RANGES } from '../../../../../../../../common/constants';
+import { useUiSetting$ } from '../../../../../../../../public/common/lib/kibana';
+
+interface Range {
+  from: string;
+  to: string;
+  display: string;
+}
 
 const DatePickerWrapper = styled.div`
   width: ${(props) => props.theme.eui.fractions.single.percentage};
-  background: white;
+  max-width: 250px;
 `;
 const StickyFlexItem = styled(EuiFlexItem)`
-  max-width: 350px;
+  background: white;
```

Comment:
you should not assume a white background due to dark mode support.  Can you use a value from the EUI styled theme?

![image](https://user-images.githubusercontent.com/56442535/129798360-e97ce0a3-3b41-4c36-9311-39f12fab555d.png)

Reply:
done 3279bd1d4477f0ad10d8c07be6874d1157fa5023